config/jobs/GoogleCloudPlatform/k8s-cluster-bundle/k8s-cluster-bundle.yaml

```diff
@@ -0,0 +1,29 @@
+presubmits:
+  GoogleCloudPlatform/k8s-cluster-bundle:
+  - name: pull-k8s-cluster-bundle-bazel-test
+    always_run: true         # Run for every PR, or only when requested.
+    decorate: true
+    labels:
+      preset-service-account: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20180918-762ed5b64-master
```

Comment:
Could we use gcr.io/cloud-builders/bazel?

https://cloud.google.com/cloud-build/docs/cloud-builders

The answer aught to be yes ;-)

Reply:
^^ address this?